microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -309,20 +286,39 @@ See also:
 * [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 120.
-* This checkbox is present in Microsoft Edge Stable 118.
+* This checkbox is present in Microsoft Edge Canary 122.
+* This checkbox is present in Microsoft Edge Stable 120.
+
+
+<!-- ====================================================================== -->
+## Disable Indentation Markers temporarily
+
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 122_
+
+Enables temporarily disabling indentation markers.
+<!-- todo: need description -->
```

Comment:
need description of the Experiments checkbox "Disable Indentation Markers temporarily"
@captainbrosset @zoherghadyali @leahmsft

Reply:
Indentation markers are vertical lines that appear in the **Sources** tool, as shown here:

<img width="425" alt="image" src="https://github.com/MicrosoftDocs/edge-developer/assets/1152698/1d2defa3-1c33-4968-a28a-7d5f3c54b4c2">

If you don't want to see them, enable the **Disable Indentation Markers temporarily** experiment, reload DevTools, and you get this instead:

<img width="436" alt="image" src="https://github.com/MicrosoftDocs/edge-developer/assets/1152698/d4c040e9-f6d2-404a-9471-48d20066a775">